web/frontend/cypress/e2e/robot-components/base.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe('should interact with base', () => {
```

Comment:
I'd recommend this language instead:
```ts
describe('base', () => {
  it('should be interactive', () => {
```
This language is a bit more idiomatic and makes tests easier to read when cypress is conducting them